boards/arm/stm32f072_eval/Kconfig.board

```diff
@@ -0,0 +1,11 @@
+# Kconfig - STM32F072-EVAL board configuration
+#
+# Copyright (c) 2017 BayLibre, SAS
+# Author: Neil Armstrong <narmstrong@baylibre.com>
```

Comment:
Remove Author line. Apply everywhere.
Or did I missed a copyright policy update in the project?

Reply:
I think it's me who missed an update since I always format my file headers like that other multiple project.
Anyway seems my previous zephyr commits didn't have the author line...

will remove !